Bindings/Packages/com.virtualmaker.bindings/Runtime/Bindings.InputActions.cs

```diff
@@ -79,14 +80,25 @@ void Canceled(T value)
                 }
             }
 
-            On<T>(inputActionReference, Performed, Started, Canceled);
+            OnInputAction<T>(inputActionReference, Performed, Started, Canceled);
         }
 
+        [Obsolete("Use 'OnInputAction' instead.", false)]
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public void On(InputActionReference inputActionReference, Action onPerformed = null, Action onStarted = null, Action onCanceled = null)
-            => On(inputActionReference, _ => onPerformed?.Invoke(), _ => onStarted?.Invoke(), _ => onCanceled?.Invoke());
+            => OnInputAction(inputActionReference, onPerformed, onStarted, onCanceled);
 
+        public void OnInputAction(InputActionReference inputActionReference, Action onPerformed = null, Action onStarted = null, Action onCanceled = null)
+            => OnInputAction(inputActionReference, _ => onPerformed?.Invoke(), _ => onStarted?.Invoke(), _ => onCanceled?.Invoke());
+
+        [Obsolete("Use 'OnInputAction' instead.", false)]
```

Comment:
I think this is just pedantic. I don't understand why we just can't use `On`.

By that logic we should also change all the other names to be specific to the input type. 

But if we're trying to generalize everything there's no point in adding the suffix of the type of input call back.